wrangler-core/src/main/java/io/cdap/directives/column/Swap.java

```diff
@@ -43,7 +43,6 @@
 @Description("Swaps the column names of two columns.")
 public class Swap implements Directive {
   public static final String NAME = "swap";
-  private static final Messages MSG = MessagesFactory.getMessages();
```

Comment:
are there imports that can be removed now?

Reply:
yea for some reason intellij did not remove unused imports on the fly for this directive. Fixed.